src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx

```diff
@@ -41,6 +42,26 @@ export default function QuestionsScreen(props: Props) {
       })
   }, [])
```

Comment:
remove the `, []` from this. not your fault but it's causing some issues w/ reloading the page